src/chocolatey.resources/helpers/functions/Install-ChocolateyPackage.ps1

```diff
@@ -220,6 +220,13 @@ functionality (see links).
 .PARAMETER IgnoredArguments
 Allows splatting with arguments that do not apply. Do not use directly.
 
+.PARAMETER BeforeInstall Script
+Specifies the commands to run after download has completed but before install steps have begun.
+Available in 0.10.16+.
+
+Use this for starting an auxilary process such as AutoHotkey, so that any timeouts are not 
+affected by the time to download.
+

```

Comment:
Just a note for consistency - we put parameter docs in the order they appear. This should later be moved ABOVE IgnoredArguments.